meeting-summary/2020-09-25-Aries.md

```diff
@@ -26,9 +26,9 @@
   - 最后,虽然结果是一致的,但是事务的执行需要时间,需要一系列的动作,在这个过程中,系统并没有保证中间状态是一致的。
 - 隔离性要求事务以隔离的方式并发地访问公共资源，彼此不产生干扰。隔离性要求系统小心地控制多个事务去访问公共资源。从一致性结果来看，事务一个一个地迭代执行,总能保证结果是正确的,但是这样性能太低。隔离性的实质是在一致性前提下如何发调度多个事务以取得更高的系统性能。
   - 要对公共资源的访问做隔离，手段无非是两个:一个是时间上隔离，要求事务按一定顺序去访问;另一个是空间上的隔离,对公共资源做多版本。直观上，空间上的隔离更具优势。
--  持久化要求一旦事务正确完成,即使出现系统故障,它的影响也会一直延续。这有两层含义:
-  -  事务完成后,它对公共资源的修改被持久化的记录,后续事务对该资源的访问都受这个事务的影响。
-  -  即使出现系统故障,也不会破坏该事务对资源的改动。
+- 持久化要求一旦事务正确完成,即使出现系统故障,它的影响也会一直延续。这有两层含义:
```

Comment:
This document contains many mathematical formulas which not be parsed by github markdown parser correctly.

Reply:
I am going to use github official latex render [service ](https://gist.github.com/a-rodin/fef3f543412d6e1ec5b6cf55bf197d7b) on this point.